framework/PostProcessors/ParetoFrontierPostProcessor.py

```diff
@@ -113,26 +138,43 @@ def run(self, inputIn):
     """
     inData = self.inputToInternal(inputIn)
     data = inData.asDataset()
-    sortedData = data.sortby(self.costID)
 
+    if self.invCost:
+      data[self.costID] = (-1.) * data[self.costID]
+    if self.invValue:
+      data[self.valueID] = (-1.) * data[self.valueID]
+
+    sortedData = data.sortby(self.costID)
     coordinates = np.zeros(1,dtype=int)
     for index,elem in enumerate(sortedData[self.costID].values):
       if (index>1) and (sortedData[self.valueID].values[index]>sortedData[self.valueID].values[coordinates[-1]]):
         # the point at index is part of the pareto frontier
-        # it is now needed to check if it satisfies the cost and/or value limits
         coordinates = np.append(coordinates,index)
 
     selection = sortedData.isel(RAVEN_sample_ID=coordinates)
+
+    if self.invCost:
+      selection[self.costID] = (-1.) * selection[self.costID]
+    if self.invValue:
+      selection[self.valueID] = (-1.) * selection[self.valueID]
+
```

Comment:
why you need to multiply (-1) again: 
`    if self.invCost:
       data[self.costID] = (-1.) * data[self.costID]
     if self.invValue:
       data[self.valueID] = (-1.) * data[self.valueID]`

Reply:
The definition of the space of the space where we want to obtain the pareto frontier is not unique (cost-value space ).
Depending on the application you can have these dimension inverted: cost vs. savings; value vs. loss of value.
This new PR address this issue so that this PP is flexible to handle several situations.